dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/task/AbstractJdbcTask.java

```diff
@@ -120,17 +120,17 @@ private void validateHeader(Class<? extends Enum<?>> headerClass, ResultSetMetaD
         resultColumnNames.add(metaData.getColumnName(i + 1));
       }
 
-      Set<String> desiredColumnNames =
+      Set<String> remainingColumnNames =
           Arrays.stream(headerClass.getEnumConstants())
               .map(
-                  enumVal -> CaseFormat.UPPER_CAMEL.to(CaseFormat.UPPER_UNDERSCORE, enumVal.name()))
+                  enumVal -> CaseFormat.UPPER_CAMEL.to(CaseFormat.LOWER_UNDERSCORE, enumVal.name()))
```

Comment:
We can skip the conversion since we compare using equalsIgnoreCase

Reply:
the conversion is mostly for `CamelCase` to `_underscore_` case. The java `equalsIgnoreCase` would be bother about the `underscores`

Reply:
I see. I suspect it might not work for all database types, then. Some databases fields are not underscored. We should make sure that all database types' cases can be compared with the Upper-Camel enum names.